README.md

```diff
@@ -30,8 +30,6 @@ The earliest supported version is currently:
 
 - Chef 13.8+
 
-### Cookbook Dependencies
```

Comment:
Please remove this change, we want to be explicit that we don't require dependencies in this cookbook :)

Reply:
looked like missing documentation rather than no dependencies :+1: